### ISSUE TITLE
Suggested Papers

### ISSUE THREAD

**Issue by cddlyf** [wvangansbeke/Self-Supervised-Learning-Overview#2]
Hi @wvangansbeke , for masked image modeling, I have two suggested papers:

[PeCo: Perceptual Codebook for BERT Pre-training of Vision Transformers](https://arxiv.org/abs/2111.12710)
[BEVT: BERT Pretraining of Video Transformers](https://arxiv.org/abs/2112.01529), CVPR 2022

**Comment by wvangansbeke** [comment]
Hi @cddlyf,

Thanks!
I will add them.
